src/test/java/com/coveo/spillway/SpillwayTest.java

```diff
@@ -384,4 +394,74 @@ public void triggersAreIgnoreIfTheStorageReturnsAnIncoherentResponse() {
 
     verify(callback, never()).trigger(any(LimitDefinition.class), any());
   }
+
+  @Test
+  public void canAddCapacityLimitOverride() {
+    LimitOverride override = LimitOverrideBuilder.of(JOHN).to(A_CAPACITY).per(A_DURATION).build();
```

Comment:
Would it be more interesting to have the LimitOverride take a Context as input instead of the Property?

You're given a user and you have to decide if this guy has an override.

Perhaps this would clash with the PropertyExtractor of the limit itself.. but what if you don't want the same throttling for users that are 18 years old vs 30 years old. You need a limit that bucketizes per age to do that, you can't use the override.